app-admin/kubelet/files/kubelet

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+# Wrapper for launching kubelet via rkt-fly stage1
+
+set -e
+
+if [ -z "${KUBELET_VERSION}" ]; then
+    echo "ERROR: must set KUBELET_VERSION"
+    exit 1
+fi
+
+KUBELET_ACI="${KUBELET_ACI:-quay.io/coreos/hyperkube}"
+
+mkdir --parents /etc/kubernetes
+mkdir --parents /var/lib/docker
+mkdir --parents /var/lib/kubelet
+mkdir --parents /run/kubelet
+
+exec /usr/bin/rkt run \
+  --volume etc-kubernetes,kind=host,source=/etc/kubernetes \
+  --volume etc-ssl-certs,kind=host,source=/usr/share/ca-certificates \
+  --volume var-lib-docker,kind=host,source=/var/lib/docker \
+  --volume var-lib-kubelet,kind=host,source=/var/lib/kubelet \
+  --volume run,kind=host,source=/run \
+  --mount volume=etc-kubernetes,target=/etc/kubernetes \
+  --mount volume=etc-ssl-certs,target=/etc/ssl/certs \
+  --mount volume=var-lib-docker,target=/var/lib/docker \
+  --mount volume=var-lib-kubelet,target=/var/lib/kubelet \
+  --mount volume=run,target=/run \
+  $RKT_OPTS \
+  --stage1-image=/usr/share/rkt/stage1-fly.aci \
+  ${KUBELET_ACI}:${KUBELET_VERSION} --exec=/hyperkube -- kubelet "$@"
```

Comment:
The state that it was left in was to build both a kubelet container and a hyperkube container, and have the bash script reference the kubelet container (while all other master components stay the same an use hyperkube).

To be honest I'm not convinced of this approach, but others felt more strongly about it (more specific footprint is mostly what I gathered). Upstream builds both a kubelet and hyperkube image FWIW.

Also, it does look like hyperkube checks argv[0] (untested):
https://github.com/kubernetes/kubernetes/blob/v1.1.2/cmd/hyperkube/main.go#L40
https://github.com/kubernetes/kubernetes/blob/v1.1.2/cmd/hyperkube/hyperkube.go#L119

If that is the main concern, I would still prefer to just be building a single image...